runtime/gc_glue_java/ScavengerBackOutScanner.cpp

```diff
@@ -61,14 +61,18 @@ MM_ScavengerBackOutScanner::scanAllSlots(MM_EnvironmentBase *env)
 	MM_RootScanner::scanAllSlots(env);
 
 	if (!_extensions->isConcurrentScavengerEnabled()) {
-	/* Back out Ownable Synchronizer Processing */
+	/* Back out Ownable Synchronizer and continuation Processing */
 		MM_HeapRegionDescriptorStandard *region = NULL;
 		GC_HeapRegionIteratorStandard regionIterator(_extensions->heapRegionManager);
 		while (NULL != (region = regionIterator.nextRegion())) {
-			MM_HeapRegionDescriptorStandardExtension *regionExtension = MM_ConfigurationDelegate::getHeapRegionDescriptorStandardExtension(env, region);
-			for (uintptr_t i = 0; i < regionExtension->_maxListIndex; i++) {
-				MM_OwnableSynchronizerObjectList *list = &regionExtension->_ownableSynchronizerObjectLists[i];
-				list->backoutList();
+			if ((MEMORY_TYPE_NEW == (region->getTypeFlags() & MEMORY_TYPE_NEW))) {
```

Comment:
put a short comment above the line like  'consistent with startProcessing that was earlier (in this GC cycle) called only on NEW regions'